examples/sdk-playground/packages/client/src/serviceWorker.ts

```diff
@@ -15,13 +15,13 @@ const isLocalhost = Boolean(
     // [::1] is the IPv6 localhost address.
     window.location.hostname === '[::1]' ||
     // 127.0.0.1/8 is considered localhost for IPv4.
-    window.location.hostname.match(/^127(?:\.(?:25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)){3}$/)
+    window.location.hostname.match(/^127(?:\.(?:25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)){3}$/),
 );
 
 export function register(config: any) {
   if (process.env.NODE_ENV === 'production' && 'serviceWorker' in navigator) {
     // The URL constructor is available in all browsers that support SW.
-    const publicUrl = new URL(process.env.PUBLIC_URL, window.location.href);
+    const publicUrl = new URL(process.env.PUBLIC_URL!, window.location.href);
```

Comment:
nit: it's not clear to me why this was necessary

Reply:
me either

Reply:
👋🏻 It's because the DTS override for `react-scripts` declares `process.env` properties as type `string` and not `string | undefined` (i.e. type [`Dict<string>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/78e6b7c1688cad1681e2efd9d50dfd7b8af49fb6/types/node/process.d.ts#L138)).

See CRA: https://github.com/facebook/create-react-app/blob/0a827f69ab0d2ee3871ba9b71350031d8a81b7ae/packages/react-scripts/lib/react-app.d.ts#L8

Reply:
lol @grant!?  So awesome to see you swing by our humble SDK!

Reply:
It's great seeing you on the interwebs, Justin! Hope we can work together again in the future :)